README.md

```diff
@@ -10,8 +10,8 @@ Learning][].
 For in-depth information on the Graph Visualizer, see this tutorial:
 [TensorBoard: Graph Visualization][].
 
-[TensorBoard: Visualizing Learning]: https://www.tensorflow.org/get_started/summaries_and_tensorboard
-[TensorBoard: Graph Visualization]: https://www.tensorflow.org/get_started/graph_viz
+[TensorBoard: Visualizing Learning]: https://www.tensorflow.org/r1/summaries
+[TensorBoard: Graph Visualization]: https://www.tensorflow.org/tensorboard/graphs
 
 You may also want to watch
 [this video tutorial][] that walks
```

Comment:
Are the APIs shown here still the right ones to show our users? I think we should overall emphasize our [new tutorials](https://www.tensorflow.org/tensorboard/get_started) and that they run in Colab.

Reply:
Removed the link to the video tutorial.